gulpfile.js

```diff
@@ -271,37 +267,27 @@ gulp.task("add-i18n", () => {
         .pipe(gulp.dest("."))
 });
 
-// Gathers all strings to Transifex readable .xliff file for translating and pushes them to Transifex
-gulp.task("transifex-push", gulp.series("build", function runTransifexPush() {
-    return gulp.src(["package.nls.json", "nls.metadata.header.json", "nls.metadata.json"])
-        .pipe(nls.createXlfFiles(transifexProjectName, transifexExtensionName))
-        .pipe(nls.pushXlfFiles(transifexApiHostname, transifexApiName, transifexApiToken));
-}));
-
-// Creates Transifex readable .xliff file and saves it locally
-gulp.task("transifex-push-local", gulp.series("build", function runTransifexPushLocal() {
+// Creates MLCP readable .xliff file and saves it locally
+gulp.task("translations-export", gulp.series("build", function runTranslationExport() {
     return gulp.src(["package.nls.json", "nls.metadata.header.json", "nls.metadata.json"])
-        .pipe(nls.createXlfFiles(transifexProjectName, transifexExtensionName))
-        .pipe(gulp.dest(path.join("..", `${transifexExtensionName}-push-test`)));
+        .pipe(nls.createXlfFiles(translationProjectName, translationExtensionName))
+        .pipe(gulp.dest(path.join("..", `${translationProjectName}-localization-export`)));
 }));
 
-// Gets the files with localized strings from Transifex
-gulp.task("transifex-pull", (done) => {
-    es.merge(defaultLanguages.map((language) => {
-        return nls.pullXlfFiles(transifexApiHostname, transifexApiName, transifexApiToken, language, [{ name: transifexExtensionName, project: transifexProjectName }])
-            .pipe(gulp.dest(`../${transifexExtensionName}-localization/${language.folderName}`))
-    }))
-        .pipe(es.wait(() => {
-            done();
-        }));
-});
-
-// Imports localization from raw localized Transifex strings to VS Code .i18n.json files
-gulp.task("i18n-import", (done) => {
+// Imports localization from raw localized MLCP strings to VS Code .i18n.json files
+gulp.task("translations-import", (done) => {
+    var options = minimist(process.argv.slice(2), {
+        string: "location",
+        default: {
+            location: "../vscode-translations-import"
+        }
+    });
     es.merge(defaultLanguages.map((language) => {
-        return gulp.src(`../${transifexExtensionName}-localization/${language.folderName}/**/*.xlf`)
+        let id = language.transifexId || language.id;
```

Comment:
Do we really need to use transifexId here?

Reply:
I'm not sure but the original [PR](https://github.com/Microsoft/vscode-chrome-debug-core/pull/390/files#diff-b9e12334e9eafd8341a6107dd98510c9R135) uses this variable. I think it would be better to leave it here until we have any updated about this variable.

Reply:
Well, okay.